### PR TITLE
Fix grammar in the documentation

### PR DESCRIPTION
**Pull request by wanderer163**

### CODE REVIEW

**Review by inancgumus — APPROVED** [review]
Thanks! LGTM.

**Comment by FloorD** [comment]
Hey @wanderer163 👋 we would love to send you a little k6 swag pack as per "thank you" for contributing to k6 this hacktoberfest. Please send your address details to floor[at]grafana.com and I'll take care it's shipped asap.

**Review by na-- — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ These tags don't show in the simple summary at the end of a k6 test (unless you
 
 ### Checks and thresholds
 
-Checks and thresholds are some of the k6 features that make it very easy to use load tests like unit and functional tests and integrate them in a CI (continuous integration) workflow.
+Checks and thresholds are some k6 features that make it very easy to use load tests like unit and functional tests and integrate them in a CI (continuous integration) workflow.
```
[the remark this answers is not in the document]
Comment (wanderer163) [reply README.md]: Reverted this.

**Review by na-- — APPROVED** [review]

**Review by codebien — APPROVED** [review]
LGTM, thanks!

**Review by yorugac — APPROVED** [review]

**Comment by yorugac** [comment]
CI failure is because of recent TLS issue which was already fixed in master.

**Pull request merged**